src/components/templates/TemplateWorkflowCard.vue

```diff
@@ -104,20 +104,63 @@ const getThumbnailUrl = (index = '') => {
       : api.apiURL(`/workflow_templates/${sourceModule}/${template.name}`)
 
   // For templates from custom nodes, multiple images is not yet supported
-  const indexSuffix = sourceModule === 'default' && index ? `-${index}` : ''
+  const indexSuffix = !!template.thumbnailVariant && index ? `-${index}` : ''
 
   return `${basePath}${indexSuffix}.${template.mediaSubtype}`
 }
 
+const thumbnailVariantRe = /.*\.(?<variant>[^\.]+)/
+const thumbnailVariantList = ['compareSlider', 'hoverDissolve', 'zoomHover']
+
+const getNameThumbnailVariant = (name: string) => {
```

Comment:
a unit test could be maybe useful on this but I didn't find a suitable test folder.
Should I add it to browser_test or create a unit_test directory ?

Reply:
Added some component testing.